superset/contrib/docker/docker-entrypoint.sh

```diff
@@ -37,7 +37,7 @@ fi
 if [ "$#" -ne 0 ]; then
     exec "$@"
 elif [ "$SUPERSET_ENV" = "development" ]; then
-    celery worker --app=superset.sql_lab:celery_app --pool=gevent -Ofair &
+    celery worker --app=superset.sql_lab:celery_app --logfile=/tmp/celery.log --loglevel=INFO --pool=gevent -Ofair &
```

Comment:
should we redirect it to stdout of the container instead?

Reply:
I would prefer to not mix it with the application's logs. But I agree that it would make logs much more accessible. What I would do is to redirect to stdout, but to make celery run in a separate container. Do you see any particular reason for which it needs to be run in the same container?

Reply:
We can consider moving celery to separate container. I don't see any disadvantages except +1 container. But let's do it in a separate issue.
Do you mind to just revert this change for now?
One of the problems I see with it - `INFO` is very verbose and I'm not sure if log rotation works correctly in this case. If not, `/tmp/celery.log` would grow very large fast and at the end would crash container with out of disk space error.

Reply:
Done [here](https://github.com/src-d/sourced-ui/pull/267/commits/2bf13a891e1d4f0e95acbb508873991bd5d5f506).

Reply:
Created #269 to discuss about moving `celery` to a separate container.